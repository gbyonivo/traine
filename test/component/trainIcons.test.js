import React from 'react';
import { create } from 'react-test-renderer';
import TrainIcon from '../../src/components/trainIcon';

const props = {
  size: 'small',
  className: 'salah'
};

describe('Train Icon', () => {
  it('should render correctly', () => {
    const actual = create(<TrainIcon {...props}/>);
    expect(actual).toMatchSnapshot();
  });
});