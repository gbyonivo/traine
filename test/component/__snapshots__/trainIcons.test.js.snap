// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Train Icon should render correctly 1`] = `
<div
  className="trainIcon undefined small"
/>
`;
